automatic/360ts/update.ps1

```diff
@@ -37,7 +37,7 @@ $version = $Matches[0]
 $url = "https://free.360totalsecurity.com/totalsecurity/${PackageName}_Setup_${version}.exe"
 
  @{
-    PackageName = $PackageName.ToLower()
+    PackageName = ( $PackageName.ToLower() )
```

Comment:
You don't need parentheses on this one Just FYI.

Reply:
I did this in the previous commit, and appveyor failed to update. So isolated the problem to being this line of change, go figure.